Makefile

```diff
@@ -96,12 +96,12 @@ run: generate fmt vet
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f config/crds
+	kubectl apply -f operator/config/crd/bases
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests
-	kubectl apply -f config/crds
-	kustomize build config/default | kubectl apply -f -
+	kubectl apply -f operator/config/crd/bases
```

Comment:
I don't get something here - isn't it enough to have the command below?

Reply:
Yeah, according to `operator/config/default/kustomization.yml`  which by the way has the `bases` deprecated field. 
Sorry I didn't look deeper I just adjusted the actual directories, I think both targets could be simplified with the following, right?:
```
install: manifests
	kustomize build operator/config/crd  | kubectl apply -f -

# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
deploy: manifests
	kustomize build operator/config/default | kubectl apply -f -
```

Reply:
yeah